src/services/explore.ts

```diff
@@ -53,3 +57,40 @@ export function saveFilter(filter: any, name: string, token?: string) {
         token
     );
 }
+
+export function fetchElementsInfos(
+    ids: UUID[],
+    elementTypes?: string[],
+    equipmentTypes?: string[]
+) {
+    console.info('Fetching elements metadata');
+
+    // Add params to Url
+    const idsParams = getRequestParamFromList(
+        ids.filter((id) => id), // filter falsy elements
+        'ids'
+    );
+
+    const equipmentTypesParams = equipmentTypes
+        ? getRequestParamFromList(equipmentTypes, 'equipmentTypes')
+        : [];
+
+    const elementTypesParams = elementTypes
+        ? getRequestParamFromList(elementTypes, 'elementTypes')
+        : [];
+
+    const params = [
+        ...idsParams,
+        ...equipmentTypesParams,
+        ...elementTypesParams,
+    ];
+
+    const urlSearchParams = new URLSearchParams(params);
+
+    const url = `${PREFIX_EXPLORE_SERVER_QUERIES}/v1/explore/elements/metadata?${urlSearchParams}`;
```

Comment:
`getRequestParamFromList()` return a `URLSearchParams`...
```suggestion
    const urlSearchParams = getRequestParamFromList([
        [ids.filter((id) => id), 'ids'], // filter falsy elements
        [equipmentTypes, 'equipmentTypes'],
        [elementTypes, 'elementTypes'],
    ]/*.filter(tuple => tuple[0])*/);

    const url = `${PREFIX_EXPLORE_SERVER_QUERIES}/v1/explore/elements/metadata?${urlSearchParams}`;
```

Or else `getRequestParamFromList(...): [string, string][]`, then:
```typescript
    const idsParams = getRequestParamFromList(
        ids.filter((id) => id), // filter falsy elements
        'ids'
    );

    const equipmentTypesParams = equipmentTypes
        ? getRequestParamFromList(equipmentTypes, 'equipmentTypes')
        : [];

    const elementTypesParams = elementTypes
        ? getRequestParamFromList(elementTypes, 'elementTypes')
        : [];

    const params = new URLSearchParams([
        ...idsParams,
        ...equipmentTypesParams,
        ...elementTypesParams,
    ]).toString();

    const url = `${PREFIX_EXPLORE_SERVER_QUERIES}/v1/explore/elements/metadata?${params}`;
```

Reply:
I went for the second option while removing the ternary operator on calls to `getRequestParamFromList`